src/core/system-logs/controllers/system-logs.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { SystemLogsService } from '../services/system-logs.service';
+import { SystemLogDto } from '../dtos/system-log.dto';
+
+@Controller('system-logs')
+export class SystemLogsController {
+  constructor(private readonly systemLogsService: SystemLogsService) {}
+
+  @Get()
+  async find(): Promise<SystemLogDto[]> {
+    return this.systemLogsService.find();
+  }
+
+  @Get(':id')
+  async get(@Param('id') id: string): Promise<SystemLogDto> {
+    return this.systemLogsService.findById(id);
```

Comment:
```suggestion
    return this.systemLogsService.get(id);
```
Jak poprawiłeś w kontrolerze to też popraw w serwisie